app/index.php

```diff
@@ -19,19 +20,73 @@
 
 $request = Request::createFromGlobals();
 
-$current_step = new Step($request->request->get("step", Step::STEP1_COLLECTING_DATA));
+$current_step = new Step($request->request->get("step", Step::STEP0_SETUP));
 
 $session = new Session();
 $session->start();
 
 
 switch ((string)$current_step) {
-    case Step::STEP1_COLLECTING_DATA:
+    case Step::STEP0_SETUP:
+        $configuration_files = array();
+        $dirs = array('/app/configurations', 'configurations');
+        foreach($dirs as $dir){
+            if (file_exists($dir))
+                $configuration_files = array_merge($configuration_files,
+                    preg_filter('/^/', $dir.'/', array_diff(scandir($dir), array('.', '..') )));
```

Comment:
Does this work on Windows? Maybe use `$dir.DIRECTORY_SEPARATOR` instead.

Reply:
Worked for me on windows with an xampp apache webserver and with the docker container, but it should be a bit cleaner with DIRECTORY_SEPARATOR, so i've no issue with that.